src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -440,7 +440,11 @@ public bool ResetCaches
         public bool EnableNodeReuse
         {
             get { return _enableNodeReuse; }
-            set { _enableNodeReuse = value; }
+            set {
+#if FEATURE_NODE_REUSE
+                _enableNodeReuse = value;
```

Comment:
I couldn't decide about that either, and eventually went this way with this justification: if we error instead of ignore here, we push the burden of changing behavior onto our consumers. If someone tries to write a build invoker that works for both of our flavors, _they_ would have to know whether the version of MSBuild they're invoking supported it. By ignoring it, we can take that responsibility for ourselves.

Convincing?